metaseq/tasks/streaming_language_modeling.py

```diff
@@ -122,17 +117,19 @@ class StreamingLanguageModelingTask(LegacyTask):
     """
 
     def __init__(self, args):
+        if not hasattr(args, "hf_tokenizer"):
+            msg = f"""`StreamingLanguageModelingTask` expects a unified tokenizer format, and not
+            the merges/vocab dual format from days of yore. You can convert the old format to the new by running
+            `python -m metaseq.scripts.unify_tokenizer <PATH_TO_TOKENIZERS>`
```

Comment:
Yeah, I thought of doing this, but backtracked because I didn't have write permission for one of the tokenizer locations on azure - can we assume that users always will?

Reply:
Maybe can try and catch the permissions error with a similar message?

Reply:
Oh we need to fix your permissions on Azure then - yeah we should assume users have write permissions to their data directories, but this needs to fail noisily (on job launch / slurm submission, before job is queued up and waiting for machines).

Reply:
Actually I think there'll be a noisy failure anyway now if the user tries to pass any of the old args (like `--vocab-filename`), the argparser will flip out as it's not expecting that arg. I don't _think_ there's a tidy way to intercept it 🤷

Reply:
It'll probably be adding some logic here: https://github.com/facebookresearch/metaseq/blob/37809ec65632c434ed722a5c8aa37cd194ce9f8b/metaseq/launcher/slurm.py#L412